app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/IntroductionFragment.kt

```diff
@@ -234,6 +241,7 @@ internal class IntroductionFragment : Fragment() {
             ui.smallGemImageView.alpha = value
             ui.createWalletContainerView.alpha = value
             ui.userAgreementAndPrivacyPolicyTextView.alpha = value
+            ui.restoreWalletCtaView.alpha = value
```

Comment:
`CTA`

Reply:
we'll need to change the xml name to have `CTA`, breaking an xml convention. are we ok with that?

Reply:
Ah, this is coming from XML, my bad sorry, let's keep it as it is.